Help/Hackathon.rst

```diff
@@ -0,0 +1,203 @@
+Hackathon Page
+==============
+
+This page intends to help the organisation of local hackathons. To start,
+grab an invitation to our Github organization. In order to do so,
+join us on coala.io/chat and say: "Hello World". We will then invite you.
+The invitation can be accepted by mail or on https://github.com/coala.
+
+.. note::
+
+  This tutorial will cover basic git functionality. If, however, you have
+  no prior knowledge, please consider reading
+  `this link <http://api.coala.io/en/latest/Developers/Git_Basics.html>`__
+  for a deeper understanding of the commands.
+
+Running coala on a local project
+--------------------------------
+
+The first thing we plan to do is let you see coala work, for yourself.
+How we plan to do this? Easily. We will simply have you run coala on a local
+project of yours to see it actually do stuff. This is probably the best way
+of you actually realizing what coala does.
+
+.. note::
+
+  If you have previously used coala and want to start contributing, you can
+  skip this section.
+
+Grabbing coala
+~~~~~~~~~~~~~~
+
+Grabbing coala is easy. Supposing you have pip installed (Python package
+manager), you can simply run:
+
+::
+
+  pip3 install coala-bears
```

Comment:
nah, i feel like we can tell them that and some other stuff from this guide right on the hackathon :) we'll use whiteboards to cover up important commands